client/index.ts

```diff
@@ -364,5 +371,55 @@ export default class PisaClient {
             .then(res => res.json())
             .then(res => res as AppointmentRequest[]);
     }
+
+    /**
+     * Backup some data to the PISA server.
+     * @param signer A function sign the PISA appointment
+     * @param data The data to backup
+     * @param customerAddress The customer address backing up the data
+     * @param startBlock The start block from when the backup should begin - should be within 5 blocks of the current block. Backup will be held for 60,000 blocks
+     * @param id The id for this backup
+     * @param nonce The version of this backup. A backup can be replaced by providing the same backup id but a greater nonce.
+     */
+    public async backUp(signer: (digest: string) => Promise<string>, customerAddress: string, data: string, startBlock: number, id: string, nonce: number) {
```

Comment:
I think I'd prefer backup instead of backUp - not very common.
Also typo in documentation: `signer A function sign the  PISA appointment`, missing "to".

Reply:
Agree, updated